packages/api-explorer/src/components/ExplorerStyle/ExplorerStyle.tsx

```diff
@@ -1,24 +1,165 @@
 /*
+
  MIT License
+
  Copyright (c) 2020 Looker Data Sciences, Inc.
+
  Permission is hereby granted, free of charge, to any person obtaining a copy
  of this software and associated documentation files (the "Software"), to deal
  in the Software without restriction, including without limitation the rights
  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
  copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions:
+
  The above copyright notice and this permission notice shall be included in all
  copies or substantial portions of the Software.
+
  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  SOFTWARE.
+
  */
 
-import { css, createGlobalStyle } from 'styled-components'
+import styled, { css, createGlobalStyle } from 'styled-components'
+import {
+  Flex,
+  IconButton,
+  Sidebar,
+  SidebarGroup,
+  SidebarItem,
+} from '@looker/components'
+
+export const HEADER_HEIGHT = '66px'
+
+export const HeaderWrapper = styled(Flex).attrs({
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  height: HEADER_HEIGHT,
+  flexDirection: 'row',
+  pt: 'medium',
+  pr: 'large',
+  pb: 'medium',
+  pl: 'large',
+})`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.ui2};
+`
+
+interface SideNavWrapperProps {
+  headerHeight?: string
+}
+
+export const SideNavToggleWrapper = styled.div<SideNavWrapperProps>`
+  position: relative;
+  margin-top: ${parseInt(HEADER_HEIGHT, 10) / 2};
+
+  ${IconButton} {
+    background: #fff;
```

Comment:
@miketorosian Can we leverage the calculated color for the theme here? Otherwise we'll have to introduce additional styling logic once we introduce additional themes.

Reply:
```suggestion
    background: ${({ theme: { colors }) => colors.background };
```

Should keep you theme-safe :)